straxen/plugins/s2cnn_processing.py

```diff
@@ -0,0 +1,154 @@
+try:
+    import strax
+    import straxen
+    straxfound=True
+except ModuleNotFoundError:
+    print("WARNING! No strax found! Strax module will not be loaded.")
+    print("Double width converter is still available. ")
+    straxfound=False
+import numpy as np
+from copy import deepcopy
+import os 
+
+class DoubleWidthConverter():
```

Comment:
I'm not sure if I understand what this class does. Since it seems to produce a map, would you say it's easy to export this map and read it in later? That would keep the code perhaps somewhat simpler, and, if you later change how this works internally you only have to update the map file. What do you think?

Reply:
It converts area per channel (unique for each event) into the format for 2D array understood by CNN (see first picture here https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:analysis:reconstruction_team:training_S2_CNN_on_WFsim) . So it has to stay as it is unique to each event.